src/components/Footer/index.tsx

```diff
@@ -55,7 +55,7 @@ export default function Footer() {
         </Cards>
         <Box textAlign='center' sx={{fontSize: 24, mt: 6}}>
           Now, how about&nbsp;
-          <Button sx={{fontSize: 24, fontWeight: 'bold'}} component={Link} href='/try-your-own-dataset?utm_content=wonder_cta'>
+          <Button sx={{fontSize: 24, fontWeight: 'bold'}} style={{ verticalAlign: 'baseline' }} component={Link} href='/try-your-own-dataset?utm_content=wonder_cta'>
```

Comment:
@634750802

Reply:
Use `sx` directly instead of `style` on this component

Reply:
> Use `sx` directly instead of `style` on this component

 <Button sx={{fontSize: 24, fontWeight: 'bold', verticalAlign: 'baseline'}}  component={Link} href='/try-your-own-dataset?utm_content=wonder_cta'>

like this？